src/ParallelRun.jl

```diff
@@ -250,7 +285,7 @@ function one_node_single_infer(config::Configs.Config,
                 unlock(results_lock)
             catch ex
                 if is_production_run || nthreads() > 1
-                    Log.exception(ex)
+                    Log.error(string(ex))
```

Comment:
`Log.exception` gives quit a bit more information. Are you sure you want to change it?